src/program/reflection.rs

```diff
@@ -345,6 +346,9 @@ pub unsafe fn reflect_attributes(ctxt: &mut CommandContext<'_>, program: Handle)
                                                 &mut attr_name_len_max);
             }
         };
+        //let's not trust the driver too much, and clamp the max_len to [63, 2047]
```

Comment:
Can you add a space between // and the text? This is more commonly used in the codebase.

Reply:
Done!